stestr/repository/file.py

```diff
@@ -135,10 +136,10 @@ def _get_test_times(self, test_ids):
         # May be too slow, but build and iterate.
         # 'c' because an existing repo may be missing a file.
         try:
-            db = dbm.open(self._path('times.dbm'), 'c')
-        except dbm.error:
+            db = my_dbm.open(self._path('times.dbm'), 'c')
```

Comment:
So I'm a bit confused by the distinction here, why do we use dbm.open() when we first create in stopTestRun() and my_dbm here?